<launch>
    <node pkg="drone_test" type="yyy_drone" name="yyy_drone" output="screen"  >
    </node>
    <node pkg="drone_test" type="star_detection" name="star_detection" output="screen" >
    </node>
    <node pkg="drone_test" type="circle" name="circle" output="screen" >
    </node>
</launch>
